docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+Try it now! The result should be similar to the example response below 👇.
 
-Here we are! The list of **restaurants** is accessible at [`http://localhost:1337/restaurants`](http://localhost:1337/restaurants).
+::: details Click me to view an example of API response
 
-::: tip CONGRATULATIONS
-👏 Congratulations, you have now completed the **Strapi Quick Start**. Where to go next?
+```json
+[
+  {
+    "id":1,
+    "name":"Biscotte Restaurant",
+    "description":"Welcome to Biscotte restaurant! Restaurant Biscotte offers a cuisine based on fresh, quality products, often local, organic when possible, and always produced by passionate producers.",
+    "published_at":"2021-05-27T15:46:43.097Z",
+    "created_at":"2021-05-27T15:40:01.290Z",
+    "updated_at":"2021-05-27T15:46:43.110Z",
+    "categories":[
+      {
+        "id":1,
+        "name":"French Food",
+        "published_at":"2021-05-27T15:46:14.704Z",
+        "created_at":"2021-05-27T15:41:59.725Z",
+        "updated_at":"2021-05-27T15:46:14.725Z"
+      },
+      {
+        "id":2,
+        "name":"Brunch",
+        "published_at":"2021-05-27T15:46:02.015Z",
+        "created_at":"2021-05-27T15:42:29.201Z",
+        "updated_at":"2021-05-27T15:46:02.035Z"
+      }
+    ]
+  }
+]
+```
+
+:::
+
+:::warning 🥳 CONGRATULATIONS!  
+Now your content is created, published, and you have permissions to request it through the API.
+Keep on creating amazing content!
+:::
+
+## ⏩  What to do next?
 
-- Consume your API with your favorite frameworks, frontend or backend programming languages just below.
-- Learn how to use Strapi with React ([Gatsby](https://strapi.io/blog/build-a-static-blog-with-gatsby-and-strapi) or [Next.js](https://strapi.io/blog/nextjs-react-hooks-strapi-food-app-1)) or Vue.js ([Nuxt.js](https://strapi.io/blog/cooking-a-deliveroo-clone-with-nuxt-vue-js-graphql-strapi-and-stripe-setup-part-1-7/)).
-- Have a look at the [Tutorials](https://strapi.io/tutorials) to deep dive into Strapi.
-- Get help on [our community forum](https://forum.strapi.io).
-- Read the [source code](https://github.com/strapi/strapi), [contribute](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md) or [give a star](https://github.com/strapi/strapi) on GitHub.
-- Follow us on [Twitter](https://twitter.com/strapijs) to get the latest news.
-- [Join the vibrant and active Strapi community](https://slack.strapi.io) on Slack.
-  :::
+### Consume your API
 
-### Consume your API using one of these technologies:
+You can learn how to consume your API with your favorite frameworks, frontend or backend programming languages by choosing one in the list below.
+
+<IntegrationLinksAlt></IntegrationLinksAlt>
+
+### Deploy your project
+
+The next step is to deploy both your Strapi backend and the frontend on the platforms of your choice.
+
+👉 We offer various deployment options for your Strapi backend — see our [Deployment guides](/developer-docs/latest/setup-deployment-guides/deployment.md).
```

Comment:
Same comment here about related content.
(see Deployment guides)

Reply:
See my [reply in this comment](https://github.com/strapi/documentation/pull/288#discussion_r642974773) Thank you Jim, waiting for Mégane's feedback on this one.